Created DiaryCard

duminduyasas42 opened the pull request.

duminduyasas42 committed:
Created DiaryCard

duminduyasas42 committed:
Created adding form

Dulajdeshan commented on src/Components/DairyCard.js:
```diff
@@ -0,0 +1,84 @@
+    const displaydisciption= readmore? description.slice(0,100):description;
+    const onClick=()=>{
+        setReadMore(!readmore);
+        console.log("ddd");
```
Remember to remove the logs before you raise a PR

Dulajdeshan commented on src/Components/DairyCard.js:
```diff
@@ -0,0 +1,84 @@
+
+
+    const [readmore,setReadMore] =useState(true);
+    const displaydisciption= readmore? description.slice(0,100):description;
```
Wrong variable declaration. This should be like `displayDescription`. And always try to use correct spellings.

Dulajdeshan commented on src/Components/DairyCard.js:
```diff
@@ -0,0 +1,84 @@
+}
+
+
+DairyCard.defaultProps={
```
This is okay. But just add empty string as default props.

Dulajdeshan commented on src/Components/DairyHome.js:
```diff
@@ -0,0 +1,49 @@
+    root: {
+      '& > *': {
+        margin: theme.spacing(1),
+        width: '25ch',
```
Use pixels instead of `ch`

Dulajdeshan reviewed (changes requested):
Go through the comments and try to fix those.

duminduyasas42 committed:
removed console.log('ddd'); from Dairy card line,
corrected the spelling to displayDescription in DairyCard,put emptry strings to the prop in DairyCard, used pixels instead of inchs in Dairy Home

Dulajdeshan commented on src/Components/DairyHome.js:
```diff
@@ -21,6 +21,7 @@ const DairyHome = () => {
         console.log("title="+title+ " description="+description);
```
Remove this log as well
